src/tox/config/loader/str_convert.py

```diff
@@ -81,13 +81,16 @@ def to_env_list(value: str) -> EnvList:
 
     @staticmethod
     def to_bool(value: str) -> bool:
-        norm = value.strip().lower()
+        norm = str(value).strip().lower()
         if norm in StrConvert.TRUTHFUL_VALUES:
             return True
         elif norm in StrConvert.FALSE_VALUES:
             return False
         else:
-            raise TypeError(f"value {value} cannot be transformed to bool, valid: {', '.join(StrConvert.VALID_BOOL)}")
+            raise TypeError(
+                f"value {value!r} cannot be transformed to bool,"
+                f" valid values: {', '.join(repr(vb) for vb in StrConvert.VALID_BOOL)}",
```

Comment:
I don't want to use repr for valid values

Reply:
When one of the valid values is the empty string, it makes for a weird message.